### PR TITLE
Fix/translate section group config

### PR DESCRIPTION
**Pull request by shibomb**
WFM
translate in Resource.php for Issue #41

Test laravel version:
``` 
            "name": "laravel/framework",
            "version": "v10.16.1",
```

### CODE REVIEW

**Comment by shibomb** [comment]
@phpsa @moskoweb 
If you decide that you can use it, please merge it.
Please review.
Best regards.

File: src/Resources/PermissionResource.php [hunk 1]
```diff
@@ -35,7 +35,9 @@ public static function getLabel(): string
 
     protected static function getNavigationGroup(): ?string
     {
-        return config('filament-authentication.section.group') ?? strval(__('filament-authentication::filament-authentication.section.group'));
+        return config('filament-authentication.section.group')
```
Comment (nguyentranchung) [inline src/Resources/PermissionResource.php]:
```php
strval(__(config('filament-authentication.section.group') ?? 'filament-authentication::filament-authentication.section.group'));
```

Comment (shibomb) [reply src/Resources/PermissionResource.php]: @nguyentranchung  Appreciate the review!

**Review by nguyentranchung — APPROVED** [review]

**Comment by shibomb** [comment]
Thanks!

**Pull request merged**